Apps learn page

corwintines opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description

Adds new learn page content for apps

https://deploy-preview-15946--ethereumorg.netlify.app/en/what-are-apps

corwintines commented:
Sorry @konopkja, think theres a clash with how the /apps/{appname} and /apps/learn page routes are being generated. This is working locally, but something happens during build. Taking a look at whats going on here.

konopkja commented:
<img width="868" height="269" alt="Screenshot 2025-07-30 at 23 41 09" src="https://github.com/user-attachments/assets/0f46edbd-0433-4e95-b7d3-4f84d1b72e85" />

this is too hard to read if this is dark mode body medium, then we should switch it just to default body color

konopkja commented:
@wackerow  "Added "Ethereum applications" to the use-cases dropdown menu since we're using that template"

i dont think it should be in that drop down, this template is something we might want to use in other areas of the website visually. Another example of this is the learn hub that doesnt have the dropdown

corwintines commented:
Addressed the comments here. @konopkja I sent you a message to sort out the stories part of this. 

Going to merge into main apps revamp branch for preview deploy purposes.
